styleguide/core/Form.hbs

```diff
@@ -0,0 +1,23 @@
+{{#defineBlock "Form"}}
+
+    {{#defineElement "actionButton"}}
+        <div class="{{elementName}}">{{this}}</div>
+    {{/defineElement}}
+
+    {{#defineElement "items"}}
+        {{#each this}}
+            <div class="{{elementName}}-item">{{this}}</div>
+        {{/each}}
+    {{/defineElement}}
+
+    {{#defineBlockContainer}}
+
+        <form class="{{blockName}}" {{#with actionUrl}} action="{{this}}" {{/with}}>
```

Comment:
Agreed there. If we do that, Mike, that will also allow us to do the demo in the styleguide via GET and do POST on the actual server for commenting